mautrix_twitter/portal.py

```diff
@@ -504,7 +506,12 @@ async def _reupload_twitter_media(
         return ReuploadedMediaInfo(mxc, decryption_info, mime_type, file_name, len(data))
 
     async def handle_twitter_reaction_add(
-        self, sender: p.Puppet, msg_id: int, reaction: ReactionKey, time: datetime
+        self,
+        sender: p.Puppet,
+        msg_id: int,
+        reaction: ReactionKey,
+        time: datetime,
+        reaction_id: int,
     ) -> None:
         async with self._reaction_lock:
             dedup_id = (msg_id, sender.twid, reaction)
```

Comment:
```suggestion
            # TODO update the database with the reaction_id of outgoing reactions
            dedup_id = (msg_id, sender.twid, reaction)
```

Probably doesn't need to be implemented now, so let's just add a todo